test/parallel/test-buffer-arraybuffer.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+
+const Buffer = require('buffer').Buffer;
+const LENGTH = 16;
+
+var ab = new ArrayBuffer(LENGTH);
```

Comment:
No idea what you're saying about fast const not being in master, but I'm fine to change `var`.